packages/linea-ccip-gateway/src/server.ts

```diff
@@ -1,16 +1,18 @@
-import { EVMGateway } from "./evm-gateway";
-import { ethers } from "ethers";
-import { L2ProofService } from "./L2ProofService";
-import "dotenv/config";
-import { Server } from "@chainlink/ccip-read-server";
-import { logError } from "./utils";
+import { Request, Response } from 'express';
+import { EVMGateway } from './evm-gateway';
+import { ethers } from 'ethers';
+import { L2ProofService } from './L2ProofService';
+import 'dotenv/config';
+import { Server } from '@chainlink/ccip-read-server';
+import { logError } from './utils';
 
 const l1ProviderUrl = process.env.L1_PROVIDER_URL;
 const l2ProviderUrl = process.env.L2_PROVIDER_URL;
-const l2ChainId = parseInt(process.env.L2_CHAIN_ID);
-const rollupAddress = process.env.L1_ROLLUP_ADDRESS;
+const l2ChainId = parseInt(process.env.L2_CHAIN_ID ?? '59144');
+const rollupAddress =
+  process.env.L1_ROLLUP_ADDRESS ?? '0xB218f8A4Bc926cF1cA7b3423c154a0D627Bdb7E5';
 const port = process.env.PORT || 3000;
-const nodeEnv = process.env.NODE_ENV || "test";
+const nodeEnv = process.env.NODE_ENV || 'test';
```

Comment:
is this meant to default to mainnet given the example was changed to testnet? 59144 vs 59141

Reply:
Yeah, it's probably better to set it to `59141`!